### PR TITLE
VoidRaptor Disposals Conveyor belt and Perma Cryo fixes

### LINKED ISSUE #15889: Void Raptor Prison lacks Cryopods
Title.

Its one of our maps so I don't see why we can't have *at least* one pod and console.

### LINKED ISSUE #15893: Void Raptor Disposals lacks a conveyor belt under the Recycler
Title.

Simple fix.
image

### PR DESCRIPTION
**Pull request by LovliestPlant**
## About The Pull Request

Fix: https://github.com/Skyrat-SS13/Skyrat-tg/issues/15893
Fix: https://github.com/Skyrat-SS13/Skyrat-tg/issues/15889

Fills in the missing conveyor belt below the recycler in VoidRaptor's disposals. And adds a cryopod and console to perma, near the showers.

## How This Contributes To The Skyrat Roleplay Experience

Disposals will be able to cycle properly, and prisoners will have the option to cryo if they so choose.

## Changelog

:cl:
add: VoidRaptor: Add a cryopod to the permabrig
fix: VoidRaptor: Resolve missing conveyor belt in disposals.
/:cl:

VR_PermaCryo

### CODE REVIEW

**Comment by Jolly-66** [comment]
If you do
Fix: #pr or Fixes: #pr
It'll automatically link the issue number and, once this PR is merged, close the issues.

**Comment by LovliestPlant** [comment]
Where abouts? In the body or in the CL?

**Comment by Jolly-66** [comment]
> Where abouts? In the body or in the CL?

The CL is for player-facing information, putting it there would be useless to those who don't use GitHub.

Put it in the PR body.

**Comment by LovliestPlant** [comment]
Okay, I think I did it right?

**Comment by ghost** [comment]
> Okay, I think I did it right?

image
All good!
It also works with "Closes #" or "Fixes #", no need for the : if you prefer to be lazy (like me)

File: _maps/map_files/VoidRaptor/VoidRaptor.dmm [hunk 1]
```diff
@@ -14285,7 +14285,6 @@
 /area/station/hallway/primary/central/fore)
 "emW" = (
 /obj/machinery/computer/cryopod{
-	dir = 8;
```
Comment (tf-4) [inline _maps/map_files/VoidRaptor/VoidRaptor.dmm:14288]: image
seems you've accidentally removed the dir on this which makes it at the wrong angle

**Review by tf-4 — CHANGES_REQUESTED** [review]

File: _maps/map_files/VoidRaptor/VoidRaptor.dmm (hunk 1 above)
Comment (LovliestPlant) [reply _maps/map_files/VoidRaptor/VoidRaptor.dmm:14288]: whoops, okay he's turned the right way round now.

**Comment by Zergspower** [comment]
probably want to avoid this going stale

**Comment by LovliestPlant** [comment]
oops. Welp. I fixed the thing.

**Review by tf-4 — APPROVED** [review]
LGTM

**Review by Paxilmaniac — APPROVED** [review]
Looks good to me

**Pull request merged**